packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -451,227 +449,178 @@ $ca-locale-ar-font-family: $ca-inter-locale-he-font-family;
   );
 }
 
-@mixin kz-typography-data-large(
-  $size: to-unitless-pixels($kz-typography-data-large-font-size),
-  $args...
-) {
+@mixin kz-typography-data-large($size: to-unitless-pixels(5.25rem), $args...) {
```

Comment:
I went through and inlined values (from the Zen theme) where CSS variables aren't supported.

I'm tossing up the idea of inlining values everywhere here, and not just where we have to for compiler happiness.
E.g. change `$typography-data-large-font-weight` to `700` within the following lines, even though the CSS variable is supported.

Reason being: we might not want some values to change dynamically (because it's a CSS variable) while others stay put.
However, this is actually happening now in its current state: when users are set to use the Heart theme, these values become out of sync (some values are from Heart theme and some are from Zen)